src/main/java/frc/trigon/robot/subsystems/shooter/Shooter.java

```diff
@@ -77,8 +77,12 @@ public boolean atTargetLeftVelocity() {
         return atVelocity(leftMotor.getSignal(TalonFXSignal.VELOCITY), targetLeftVelocityRotationsPerSecond);
     }
 
-    public boolean atVelocity(double currentVelocty, double targetVelocity) {
-        return Math.abs(currentVelocty - targetVelocity) < ShooterConstants.VELOCITY_TOLERANCE;
+    void shootCloseShot() {
+        setTargetVelocity(ShooterConstants.CLOSE_SHOT_VELOCITY_ROTATIONS_PER_SECOND, ShooterConstants.CLOSE_SHOT_VELOCITY_ROTATIONS_PER_SECOND * ShooterConstants.RIGHT_MOTOR_TO_LEFT_MOTOR_RATIO);
+    }
+
+    void shootToAmp() {
+        setTargetVelocity(ShooterConstants.AMP_SHOOTING_VELOCITY_ROTATIONS_PER_SECOND, ShooterConstants.AMP_SHOOTING_VELOCITY_ROTATIONS_PER_SECOND);
     }
```

Comment:
Honestly, instead of having these two, I would just call `getSetTargetVelocityCommand` with a constant velocity.